docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,172 @@
+
+To install the Fabric CA client, which is used to register and enroll identities, follow the instructions in [the Fabric CA User's Guide](https://hyperledger-fabric-ca.readthedocs.io/en/latest/users-guide.html#overview). Note that there are a number of configuration options when deploying a CA (the database type, for example, or whether to use LDAP).
+
+For an example of how to setup a CA and enroll its admin, check out [Setup Orderer Org CA](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#setup-orderer-org-ca). For an example of how to set up a TLS CA, check out [Setup TLS CA](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#setup-tls-ca). Note that bootstrapping the CA server means assigning a username and password that functions as "registering" the CA admin.
+
+### Step four: Use the CA to create identities and MSPs
+
+After you have created your CAs, you can use them to create the certificates for the identities and components related to your organization (which is represented by an MSP). For each organization, you will need to, at a minimum:
+
+* **Register and enroll an admin identity and create an MSP**. After the CA that will be associated with an organization has been created, it can be used to first register an identity and then enroll it. In the first step, a username and password for the identity is assigned by the admin of the CA. Attributes and affiliations can also be given to the identity (for example, a `role` of `admin`, which is necessary for organization admins). After the identity has been registered, it can be enrolled by using the username and password. The CA will generate two certificates for this identity --- a public certificate (also known as a signcert) known to the other members of the network, and a private certificate (stored in the `keystore` folder) used to sign actions taken by the identity. The CA will also generate an MSP file containing the public certificate of the CA issuing the certificate and the root of trust for the CA (this may or may not be the same CA). This MSP can be thought of as defining the organization associated with the identity of the admin. For an example of how this process looks, check out the [this example of how an admin is enrolled](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin). In cases where the admin of the org will also be an admin of a node (which will be typical), **you must create the org admin identity before creating the local MSP of a node, since the certificate of the node admin must be used when creating the local MSP**.
```

Comment:
I wouldn't say it's a private certificate that is stored in the `keystore` folder. It's the actual private key that is stored there, the certificate contains the public key. When you issue a command to the CA to generate the cert, the client keeps the private key and sends the public key to CA to get signed.